pinakes/main/migrations/0045_source_last_refresh_stats.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.5 on 2022-05-05 15:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("main", "0044_alter_portfolioitem_name_alter_serviceoffering_name"),
```

Comment:
The migration conflicts with #504 and #506. Depends on which RP gets merged first, the other two need rename.